ext/fg/js/popup.js

```diff
@@ -79,6 +76,12 @@ class Popup {
 
     // Public functions
 
+    prepare() {
+        this._container.addEventListener('mousedown', (e) => e.stopPropagation());
+        this._container.addEventListener('scroll', (e) => e.stopPropagation());
+        this._container.addEventListener('load', this._onFrameLoad.bind(this));
+    }
```

Comment:
It doesn't have side effects either way, but I'm fine with it being in prepare.
62be1f39da08cfa2e2a8589089f3552a28c465ef

Reply:
```javascript
this._container.style.setProperty('visibility', this.isVisibleSync() ? 'visible' : 'hidden', 'important');
```

Not sure if that should be considered a side effect, but not a big deal. I think it makes sense in prepare because it's a function call.

Reply:
Since the node isn't added to a document, I wouldn't consider it one. Some style properties are already assigned in the constructor anyway.